mmv1/third_party/terraform/resources/resource_sql_database_instance.go.erb

```diff
@@ -1070,6 +1075,7 @@ func expandLocationPreference(configured []interface{}) *sqladmin.LocationPrefer
 	return &sqladmin.LocationPreference{
 		FollowGaeApplication: _locationPreference["follow_gae_application"].(string),
 		Zone:                 _locationPreference["zone"].(string),
+	    SecondaryZone:        _locationPreference["secondary_zone"].(string),
```

Comment:
nit: I think the spacing here is inconsistent- can you ensure it's using tabs and lines up w/ above?

Reply:
fixed!